time/src/lib.rs

```diff
@@ -0,0 +1,9 @@
+#[cfg(target_family = "wasm")]
+mod gloo;
+#[cfg(not(target_family = "wasm"))]
+mod tokio;
+
+#[cfg(target_family = "wasm")]
+pub use gloo::*;
+#[cfg(not(target_family = "wasm"))]
```

Comment:
I'd do it differently by putting this into the `Cargo.toml` such that these dependencies get included only depending on the target. For this to work, the `Cargo.toml` should look like this:
```
[dependencies]
[target.'cfg(target_arch = "wasm32")'.dependencies]
gloo-timers = { version = "0.2", features = ["futures"], optional = true }
[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
tokio = { version = "1.36", features = ["time"], optional = true }
[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
tokio-stream = { version = "0.1", features = ["time"], optional = true }
```
Then the code here could just use the features `tokio`, `tokio-stream` and `gloo` for enabling code here.